.circleci/config.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           name: Run tests
           command:  |
             source activate py37_env
-            py.test nimare
+            py.test -m "not performance" nimare
```

Comment:
We may want a proper Makefile to manage our tests. Something like [this one](https://github.com/Brainhack-Donostia/aroma/blob/bcca026a9a270ecf11a38d32b3967ac72bf49a81/Makefile). WDYT? Do you have a preference on how to split tests?

Reply:
Perhaps `pytest.mark` is a better choice... TBH I'm not familiar with it.

Reply:
We can use both a makefile and `pytest mark`. `pytest.mark` replaces the `--skipintegration`, `-k` syntax when calling pytest, and the `if skip_integration:` logic in the integration tests, I honestly haven't used marks a lot so I do not have a strong preference, but from my little experience, marks are relatively easy to use, and I can add multiple marks to a single test function to give finer control on what tests should be run.
I'll get a makefile made.